node/engine/chainservice/chainservice.go

```diff
@@ -66,13 +68,41 @@ type ConcludedEvent struct {
 	commonEvent
 }
 
-type ChallengeEvent struct {
+func (ce ConcludedEvent) String() string {
+	return "Channel " + ce.channelID.String() + " concluded at Block " + fmt.Sprint(ce.blockNum)
+}
+
+type ChallengeRegisteredEvent struct {
 	commonEvent
-	// TODO fill out other fields
+	candidate           state.VariablePart
+	candidateSignatures []state.Signature
 }
 
-func (ce ConcludedEvent) String() string {
-	return "Channel " + ce.channelID.String() + " concluded at Block " + fmt.Sprint(ce.blockNum)
+// StateHash returns the statehash which will have been stored on chain in the adjudicator after the ChallengeRegistered Event fires.
```

Comment:
The tense of these messages is a little hard to parse. Not sure if the below is better.

```suggestion
// StateHash returns the statehash stored on chain at the time of the ChallengeRegistered Event firing.
```